components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -514,9 +514,14 @@ - (BOOL)fhv_shouldCollapseToStatusBar {
 }
 
 - (BOOL)fhv_canShiftOffscreen {
+#if TARGET_OS_TV
```

Comment:
If possible have the if clause be true in the common case.  In our case, most clients will be iOS so:

#if TARGET_OS_IOS
// Common flow
#else if TARGET_OS_TV  // Don't just have an #else clause, the TARGET_OS_TV is more explicit.
// Less common flow
#endif